_guide/_pages/interviews/core-values.md

```diff
@@ -54,108 +54,61 @@ Begin by introducing yourself by saying this or something similar to it:
+- [Have you worked directly with customers/clients before?](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.8w4325mhjtnd)
     - If applicable: Could you speak about a time when you dealt with a particularly difficult client?
 
-- [Talk to me about how you prioritize features.](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
-    - How do you balance requests from various stakeholders, especially when their priority is different than yours?
+- [Tell me about your experience working on teams that directly involved non-engineers. How did you approach working with others with different professional skills?](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.6qssa3tfp2ht)
+    - How did you resolve any conflicts that arose?
+    - How did you incorporate other perspectives into your technical choices, especially when that might lead you to a different technical strategy?
 
-- [Have you had a chance to participate in user-centered design and/or user research?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
-    - If yes: Could you speak about a time when business needs or management directives conflicted with executing on a user-centered design approach?
-    - If yes: How did you overcome this challenge and what was the resolution?
-    - If no: What does user-centered design mean to you?
-
-- [Could you describe your experience with agile software development, if you have any?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
-    - If yes: Were you actively involved in helping your team/department/organization transition to agile?
-        - What challenges did you face, if any, and how did you overcome them?
-    - If no: What does agile software development mean to you?
 
 ### Communication and collaboration
 
 Following a discussion around a candidate’s relevant work experience, transition into a more focused dialogue around communication itself.
 
 Ask one (or more) of these questions, as appropriate:
 
-- [Have you ever worked with a remote/distributed team before?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
-    - If yes: What tools and/or mechanisms did you use to help promote open dialogue and foster communication between yourself and your teammates?
-    - If yes: How did you overcome any communication challenges and barriers?
-    - If no: what challenges do you anticipate? What do you think you'll need to be successful?
-
-- [Have you ever faced any challenges with a coworker or manager?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
-    - If applicable: How did you resolve the matter?
```

Comment:
I find myself missing this question (or a similar one) in the updates. I found it useful to uncover how the candidate approaches interpersonal conflict resolution.